scripts/eval_calc_errors.py

```diff
@@ -213,9 +220,7 @@
     logger.info("Organizing estimation targets...")
     targets_org = {}
     for target in targets:
-        targets_org.setdefault(target["scene_id"], {}).setdefault(target["im_id"], {})[
-            target["obj_id"]
-        ] = target
+        targets_org.setdefault(target["scene_id"], {})[target["im_id"]] = target
```

Comment:
I understand that this change is made to accommodate the fact that `test_targets_bop24.json` do not have "obj_id" keys but for `test_targets_bop19.json` this loses the information (you could put `= None`, or targets_org could simply be {<scene_id1>: [<im_id1>, <im_id2>...]} structured dictionrry, it would be the same.
I think instead we should check whether "obj_id" is in each target (sign of a bop19/localization target), fill `targets_org` as before and if not put a None value otherwise (bop24/detection target type).